pilot/pkg/model/authentication.go

```diff
@@ -83,11 +83,6 @@ func initAuthenticationPolicies(env *Environment) *AuthenticationPolicies {
 
 func (policy *AuthenticationPolicies) addRequestAuthentication(configs []Config) {
 	for _, config := range configs {
-		if config.Namespace == policy.rootNamespace && config.Name != "default" {
```

Comment:
Does this mean after this we can create a global `RequestAuthentication` with any name?

Reply:
What if two configs created?

Reply:
That's right, you can have more than one global RequestAuthentication. They will be `selected` and `merged` in the same way as in any other regular namespace.